### DISCUSSION TITLE
高分辨率下某些关卡会划过头

### DISCUSSION THREAD

**Discussion by cyscloud1015** [MaaAssistantArknights/MaaAssistantArknights#5602] [Q&A]
例如绿野幻梦的DV-6，进入关卡导航界面之后会划过头，导致选不到指定的关卡，然后一直卡在这个界面鬼畜，如视频所示。蹲个大佬看看，感谢！

https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/90675990/c36a21b7-3265-4fd4-974d-6f3411e9139a

**Comment by ABA2396** [comment]
#5228

**Reply by cyscloud1015** [reply]
get，非常感谢
